tools/linter_lib/custom_check.py

```diff
@@ -325,12 +325,6 @@
             "good_lines": ["id = alice.user_id"],
             "bad_lines": ["id = alice.userid"],
         },
-        {
-            "pattern": r"json_success\({}\)",
-            "description": "Use json_success() to return nothing",
-            "good_lines": ["return json_success()"],
-            "bad_lines": ["return json_success({})"],
-        },
```

Comment:
I think we could reasonably change this to ban `json_success(request, {})` instead, but I think that realistically, nobody is going to type that, so yeah, we can remove this.

Reply:
Agreed. Interestingly, I did find at least one instance of `data={}` and a few instances of `{"type": "success", "msg": ""}` in the search/replace commit that I removed, perhaps other small things that were missed in previous refactors.